doc/changelog.rst

```diff
@@ -0,0 +1,303 @@
+Shoop Change Log
+================
+
+Unreleased
+----------
+
+- List all changes after last release here (newer on top).  Each change on a
+  separate bullet point line.  Wrap the file at 79 columns or so.  When
+  releasing next version, the "Unreleased" header will be replaced with
+  appropriate version header and this help text will be removed.
+
+Core
+~~~~
+
+- API and documentation clean-up
+- Remove ``AddressManager``
+- Split address into mutable and immutable address
+- Add Product.get_public_media helper function
+- Ensure ``TranslatabelShoopModel.__str__`` doesn't return lazy objects
+- Deny price unit changes of in-use ``Shop``
+- Assign created customers to ``CustomerTaxGroups`` on creation
+- Fix couple tax related terms
+- Remove ``PriceTaxContext``
+- Add ``PricingContextable`` interface and fix related docstrings
+- Remove ``Priceful.total_price``
+- Add dynamic configuration API: ``shoop.configuration``
+- Fix tax calculations and implement override groups
+- Add autoexpiring versioned cache API: ``shoop.core.cache``
+- Enable email login and password recovery with username
+
+Localization
+~~~~~~~~~~~~
+
+- Add translations for Finnish, Chinese and Japanese
+- Add translation extraction tools (``shoop_makemessages``)
+- Mark more messages for translation in templates
+- Enable JavaScript translations
+
+Admin
+~~~~~
+
+- Show Shoop version number in Admin
+- Fix order list sorting and filtering by total price
+- Fix CMS page list sorting by title
+- JavaScript compilation fixes: Turn our ES6 to ES5 with Babel
+- Fix URL encoding in ``redirect_to_login``
+- Add view for creating orders from the Admin
+- Enable markdown editor for product and category description
+- Fix SKU and name initialization when creating a product by search
+- Add new template macros
+- Refactor templates to use template macros
+- Admin form styling and UX updates
+
+Front
+~~~~~
+
+- Set default country in checkout address forms
+- Fix SHOOP_FRONT_INSTALL_ERROR_HANDLERS setting being not respected
+- Change password recover error message
+
+Xtheme
+~~~~~~
+
+- Make Xtheme plugins translatable
+- Allow addons to inject resources
+- Editor improvements
+- Fix a crash when trying to revert unsaved configuration
+
+Classic Gray Theme
+~~~~~~~~~~~~~~~~~~
+
+- Show product media at order history and product detail pages
+- Add language changer to navigation
+- Add possibility for other future brand colors
+- Add carousel styles for Bootstrap carousel
+- Unvendor fonts
+- Show maintenance mode for super user
+- Fix logo text line height
+- Add new placeholders
+- Footer CMS Pages field are no longer required
+- Update label for footer links to avoid confusion
+
+Default Theme
+~~~~~~~~~~~~~
+
+- Remove Default theme from Shoop Base. Moved to
+  https://github.com/shoopio/shoop-simple-theme
+
+General/miscellaneous
```

Comment:
And by end users you mean who?
* (D) Developers who are using Shoop for their project?
* (A) Shoop web shop administrators (who set-up Shoop for someone)?
* (M) Shoop web shop merchants (who sell stuff with Shoop)?
* (C)  Customers buying from Shoop web shop?

The change log is intended for D and A. Release notes is for D, A and M.